### DISCUSSION TITLE
如何拥有一张海外卡，用于注册Claude和Twitter | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#12] [General]
# 如何拥有一张海外卡，用于注册Claude和Twitter | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/blogs/get-a-oversea-sim

**Comment by rubickecho** [comment]
用英国海外卡注册 Claude 后，需要保持英国节点访问 Claude 吗？

**Reply by weijunext** [reply]
不用，没要求登录地址要跟手机号归属地一致。不过有人建议节点要稳定。

**Reply by rubickecho** [reply]
好的谢谢呢

**Comment by howjohn** [comment]
请问我的giffgaff英国卡在绑定x的时候，提示无法向此号码发送信息，不支持的运营商。 我选的是+44 uk

**Reply by howjohn** [reply]
电脑端不行，在安卓手机端可以

**Reply by weijunext** [reply]
推特的问题。我刚开始也这样，过了几个小时又可以了。
